sql/pgtap.sql.in

```diff
@@ -2418,6 +2418,18 @@ RETURNS TEXT AS $$
     SELECT col_has_check( $1, $2, 'Column ' || quote_ident($1) || '(' || quote_ident($2) || ') should have a check constraint' );
 $$ LANGUAGE sql;
 
+-- col_has_exclusion(schema, table, columns, description)
+CREATE OR REPLACE FUNCTION col_has_exclusion(TEXT, TEXT, TEXT[], TEXT)
+RETURNS TEXT AS $$
+    SELECT ok(array_agg(attr.attname)::TEXT[] @> $3 AND $3 @> array_agg(attr.attname)::TEXT[])
```

Comment:
Looks like the `@>` operator will work on [Postgres 9.1](https://www.postgresql.org/docs/9.1/functions-array.html). Would you add tests for these new functions?